src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -82,16 +98,18 @@ export class InstalledPlugin extends React.Component<Props> {
     });
   };
 
-  enablePlugin = async () => {
+  toggleEnablePlugin = async (projectId: string, status: boolean = true) => {
     try {
       addLoadingMessage(t('Enabling...'));
-      await this.pluginUpdate({enabled: true});
-      addSuccessMessage(t('Configuration was enabled'));
-      this.props.onEnablePlugin(this.projectId);
+      await this.updatePluginEnableStatus(status);
+      addSuccessMessage(
+        tct('Configuration was [action].', {action: status ? 'enabled' : 'disabled'})
+      );
+      this.props.onEnablePlugin(projectId, status);
       this.props.trackIntegrationEvent({
         eventKey: 'integrations.enabled',
         eventName: 'Integrations: Enabled',
```

Comment:
@NisanthanNanthakumar Need to update the analytics event if the action is to disable it.